src/compas/geometry/primitives/vector.py

```diff
@@ -676,6 +679,39 @@ def unitized(self):
         v.unitize()
         return v
 
+    def invert(self):
+        """Invert the direction of this vector
+
+        Note
+        ____
+        a negation of a vector is similar to inverting a vector
+
+        Examples
+        --------
+        >>> u = Vector(1.0, 0.0, 0.0)
+        >>> v = u.copy()
+        >>> u.invert()
```

Comment:
perhaps add here
```python
>>> u == v
False
```
otherwise the test is also correct if basically nothing happens :)

Reply:
nice catch!